examples/bridge-app/linux/main.cpp

```diff
@@ -478,6 +549,44 @@ EmberAfStatus HandleReadSwitchAttribute(DeviceSwitch * dev, chip::AttributeId at
     return EMBER_ZCL_STATUS_SUCCESS;
 }
 
+EmberAfStatus HandleReadPowerSourceAttribute(DevicePowerSource * dev, chip::AttributeId attributeId, uint8_t * buffer,
+                                             uint16_t maxReadLength)
+{
+    if ((attributeId == ZCL_POWER_SOURCE_BAT_CHARGE_LEVEL_ATTRIBUTE_ID) && (maxReadLength == 1))
+    {
+        *buffer = dev->GetBatChargeLevel();
+    }
+    else if ((attributeId == ZCL_POWER_SOURCE_ORDER_ATTRIBUTE_ID) && (maxReadLength == 1))
+    {
+        *buffer = dev->GetOrder();
+    }
+    else if ((attributeId == ZCL_POWER_SOURCE_STATUS_ATTRIBUTE_ID) && (maxReadLength == 1))
+    {
+        *buffer = dev->GetStatus();
+    }
+    else if ((attributeId == ZCL_POWER_SOURCE_DESCRIPTION_ATTRIBUTE_ID) && (maxReadLength == kPowerSourceDescriptionSize))
+    {
+        uint8_t bufferMemory[kPowerSourceDescriptionSize];
+        MutableByteSpan zclString(bufferMemory);
+        MakeZclCharString(zclString, dev->GetDescription().c_str());
+        buffer = zclString.data();
```

Comment:
This part doesn't make sense to me.  Was this tested?  It's certainly not copying the string into the _caller_'s buffer...

Reply:
So I just copy paste from the node label attribute reading handle. 
Now it's fixed and tested.

Reply:
> So I just copy paste from the node label attribute reading handle.

@AntonGrey File a followup to get that fixed as needed?

Reply:
I'll file an issue a bit later.